src/components/Icons/Close/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+export const Icon = styled.svg`
+  cursor: pointer;
+`;
+
+const CloseIcon = ({
+  size,
+  color,
+  className,
+  onClick,
+}) => (
+  <Icon
+    xmlns="http://www.w3.org/2000/svg"
+    xmlnsXlink="http://www.w3.org/1999/xlink"
+    width={size}
```

Comment:
Talvez seja válido acrescentar, e.g `size || 30px`, caso quem for utiilzar passar `null || undefined`